src/patternfly/components/Table/examples/Table.md

```diff
@@ -2168,12 +2168,279 @@ By default, truncation and wrapping settings do not affect the grid layout, but
 {{/table}}
 ```
 
-### Usage
+### Favorites
+```hbs
+{{#> table table--id="table-favorites" table--grid="true" table--modifier="pf-m-grid-md" table--attribute='aria-label="This is a favorites table example"'}}
+  {{#> table-thead}}
+    {{#> table-tr}}
+      {{#> table-td table-td--check="true"}}
+        <input type="checkbox" name="{{table--id}}-check-all" aria-label="Select all rows">
+      {{/table-td}}
+      {{> table-td table-td--IsEmpty="true"}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Repositories
+      {{/table-th}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Branches
+      {{/table-th}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Pull requests
+      {{/table-th}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Workspaces
+      {{/table-th}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Last commit
+      {{/table-th}}
+      {{> table-td table-td--IsEmpty="true"}}
+    {{/table-tr}}
+  {{/table-thead}}
 
-| Class | Applied to | Outcome |
+  {{#> table-tbody}}
+    {{#> table-tr table-tr--id="1"}}
+      {{#> table-td table-td--check="true"}}
+        <input type="checkbox" name="{{table--id}}-checkrow{{table-tr--id}}" aria-labelledby="{{concat table--id '-node' table-tr--id}}">
+      {{/table-td}}
+      {{> table-td table-td--favorite="true" table-td--IsFavorited="true"}}
+      {{#> table-td table-td--data-label="Repository name"}}
```

Comment:
Looks like we need a text wrapper `{{#> table-td--text}}`

![Screen Shot 2020-10-20 at 5 00 39 PM](https://user-images.githubusercontent.com/5385435/96643767-10520e80-12f6-11eb-8763-91d6828e0efa.png)

Reply:
Doh! Good catch. Just wrapped that in a `<div>` since that's what we're doing in most of the other examples with cells that have elements.